### PR TITLE
add scriptTimeout option + fix issue #18

### PR DESCRIPTION
**Pull request by okeul**
Related to https://github.com/felixzapata/gulp-axe-webdriver/issues/14#issuecomment-409875369

ping @ryuran

### CODE REVIEW

**Review by ryuran — APPROVED** [review]

**Review by ryuran — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -135,6 +141,13 @@ Default value: ''
 
 An optional file to which the results of the accessibility scans will be written as a JSON Array of results objects.
 
+#### scriptTimeout
+Type: `Number`
+
+Default value: `null`
```
Comment (felixzapata) [inline README.md]: Should be the devault value at least 60000 or less instead of null?

**Comment by okeul** [comment]
@felixzapata Updated with your previous comment about default value of scriptTimeout

**Comment by felixzapata** [comment]
@okeul can you take a look to the tests within this branch?

`npm run test`

I obtain this error:

captura de pantalla 2018-09-18 a las 11 42 39

**Comment by okeul** [comment]
@felixzapata Good catch ;-) It fixed.

**Comment by felixzapata** [comment]
thanks @okeul for the pull request.

**Pull request merged**